plugins/emmc/fu-emmc-device.c

```diff
@@ -525,9 +541,27 @@ fu_emmc_device_write_firmware(FuDevice *device,
 	}
 	fu_progress_step_done(progress);
 
+	g_debug("programming success");
+
```

Comment:
is this really needed?  it's only shown in `--verbose --verbose`, and never changes.  I'd think the plugin progress hitting this step done is _probably_ enough.

Reply:
thanks.  will remove.